bot/converters.py

```diff
@@ -0,0 +1,30 @@
+from discord.ext.commands import BadArgument, Context, Converter
+
+
+class OffTopicName(Converter):
+    """
+    A converter that ensures an added off-topic name is valid.
+
+    Taken from python-discord/bot https://github.com/python-discord/bot/blob/master/bot/converters.py#L344
```

Comment:
```suggestion
    Adapted from python-discord/bot https://github.com/python-discord/bot/blob/master/bot/converters.py#L344
```
This might be more fitting

Reply:
This should also mention the license and copyright

Reply:
in any specific format?